packages/babel-code-frame/src/index.js

```diff
@@ -7,16 +7,18 @@ import chalk from "chalk";
  */
 
 let defs = {
-  string:     chalk.red,
-  punctuator: chalk.bold,
-  curly:      chalk.green,
-  parens:     chalk.blue.bold,
-  square:     chalk.yellow,
-  keyword:    chalk.cyan,
-  number:     chalk.magenta,
-  regex:      chalk.magenta,
-  comment:    chalk.grey,
-  invalid:    chalk.inverse
+  keyword:     chalk.yellow.bold,
+  capitalized: chalk.yellow,
+  jsx_tag:     chalk.yellow.dim,
+  punctuator:  chalk.yellow.dim,
+  // bracket:  intentionally omitted.
+  number:      chalk.yellow,
+  string:      chalk.yellow,
+  regex:       chalk.yellow,
+  comment:     chalk.grey,
+  invalid:     chalk.white.bgRed.bold,
+  gutter:      chalk.grey,
+  marker:      chalk.red.bold,
```

Comment:
I just used a babel project, wrote the code you pasted before and then ran it by editing node_modules